_api-reference/validate.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: default
+title: Validate Query
+nav_order: 87
+---
+
+# Validate Query
+
+You can use the Validate Query API to validate a query without running it. The query can be sent as a path parameter or in the request body.
+
+## Path and HTTP methods
+
+The Validate Query API contains the following paths:
+
+```
+GET <index>/_validate/query
+```
+
+## Path parameters
+
+All path parameters are optional.
+
+Parameter | Data type | Description
+:--- | :--- | :---
+`index` | String | The index to validate the query against. If you don't specify an index or multiple indexes as part of the URL (or want to override the URL value for an individual search), you can include it here. Examples include `"logs-*"` and `["my-store", "sample_data_ecommerce"]`.
+`query` | Query object | The query using [Query DSL]({{site.url}}{{site.baseurl}}/query-dsl/).
+
+## Query parameters
+
+The following table lists the available query parameters. All query parameters are optional.
+
+Parameter | Data type | Description
+:--- | :--- | :---
+`all_shards` | Boolean | When `true`, validation is run against all shards instead of one shard per index. Default is `false`.
```

Comment:
```suggestion
`all_shards` | Boolean | When `true`, validation is run against [all shards](#rewrite-and-all_shards) instead of one shard per index. Default is `false`.
```